src/plugins/type/README.md

```diff
@@ -22,6 +22,10 @@ The type checker plugin supports all basic CORBA types:
 `octet`. When checking `any` it will always be successful, regardless
 of the content.
 
+The metadata `check/type` can be used to override the `type` metadata.
+This can be useful for code generation. In most cases though, `type` will
```

Comment:
"This can be useful for code generation." -> "This can be useful if the type to check differs to the type for code generation."

Reply:
"... code generation or the highlevel API"

The highlevel API also doesn't use `check/type`.